src/OVHSwiftAdapter.php

```diff
@@ -127,6 +127,48 @@ public function getTemporaryUrl(string $path, \DateTimeInterface $expiresAt, arr
         );
     }
 
+    public function getFormPostSignature(string $path, array $options = []): string
```

Comment:
I was thinking from the usage point of of view that if I only need to change the redirect, then I would need to still specify the other two parameters.

But then from an IDE perspective, what you're suggesting is much better.

Reply:
I'm just assuming redirects will not be a mayor use case, but we could just move its signature to be before file options...

```suggestion
    public function getFormPostSignature(string $path, \DateTimeInterface $expiresAt, ?string $redirect = null, int $maxFileCount = 1, int $maxFileSize = 26214400): string
```

This way, we can still skip the redirect with a "known" value, and let the developer focus on changing the following parameters.
Thinking about it, I think this would be a better order, that way, we keep the file size limit set to 25 MB by default.

Reply:
Fair enough! The documents mention that this must be empty string if no redirect is desired. Will give it a check with null... and if it doesn't work I'll change the default value to empty.